src/sentry/api/endpoints/integrations/sentry_apps/components.py

```diff
@@ -54,23 +55,29 @@ def get(
         components = []
         errors = []
 
-        for install in SentryAppInstallation.objects.get_installed_for_organization(
-            organization.id
-        ).order_by("pk"):
-            _components = SentryAppComponent.objects.filter(
-                sentry_app_id=install.sentry_app_id
-            ).order_by("pk")
+        with sentry_sdk.start_transaction(name="sentry.api.sentry_app_components.get"):
```

Comment:
Is just adding a span here sufficient? The docs on performance lead me to believe we already have a top-level transaction via the SDK for Django: https://docs.sentry.io/platforms/python/tracing/instrumentation/automatic-instrumentation/

Reply:
<img width="486" alt="Screenshot 2024-09-12 at 10 35 58" src="https://github.com/user-attachments/assets/f0e8ab0d-5113-4fa6-92c2-e84031b27ad9">

The in app instructions are telling me to add a transaction too

Reply:
Ah missed that, thanks!